debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/RowMapper.java

```diff
@@ -190,6 +191,26 @@ public static String getSqlRedo(OracleStreamingChangeEventSourceMetrics streaing
         return result.toString();
     }
 
+    public static String getRowId(OracleStreamingChangeEventSourceMetrics streamingMetrics, ResultSet rs) {
+        try {
+            return rs.getString(ROW_ID);
+        }
+        catch (SQLException e) {
+            logError(streamingMetrics, e, "ROW_ID");
+            return null;
+        }
+    }
+
+    public static int getRollbackFlag(OracleStreamingChangeEventSourceMetrics streamingMetrics, ResultSet rs) {
+        try {
+            return rs.getInt(ROLLBACK_FLAG);
+        }
+        catch (SQLException e) {
+            logError(streamingMetrics, e, "ROLLBACK");
+            return 0;
+        }
```

Comment:
Should this rather be bubbled up? Same for `getRowId()`?

Reply:
I was actually looking at these as part of the work I've been doing with CLOB support.  I believe we have a few places where we likely should bubble up these exceptions rather than log & ignore.  I suppose I can make those changes here or we can circle back and fix these separately, your call.

Reply:
I would prefer to do "the right thing (TM)" in new/changed code like this, i.e. not adding more usages of the bad pattern, if we already know we want to have it done otherwise. Rework of other, unrelated occurrences could be done separately, if this is tracked somewhere.

Reply:
Sure, I'll add a commit that bubbles this up in this PR.